builtin/tag.c

```diff
@@ -106,7 +106,22 @@ static int delete_tag(const char *name, const char *ref,
 static int verify_tag(const char *name, const char *ref,
 				const unsigned char *sha1)
```

Comment:
Yeah, I thought about that too, also about the docs/man pages.
No, we haven't yet.
Should we do this right away?